support-frontend/app/controllers/SubscriptionsController.scala

```diff
@@ -82,7 +82,7 @@ class SubscriptionsController(
     val digitalSubscription = service
       .getPrices(
         DigitalPack,
-        List(DefaultPromotions.DigitalSubscription.Monthly.fiftyPercentOff3Months),
+        Nil,
```

Comment:
Why are we changing this in this PR?

Reply:
I just changed it as support-models/src/main/scala/com/gu/support/promotions/DefaultPromotions.scala hasn't got any value for DigitalSubscription